cassandra_migrate/migrator.py

```diff
@@ -419,7 +425,9 @@ def _apply_python_migration(self, version, migration):
         self.logger.info('Applying python script')
 
         try:
-            check_call([sys.executable, migration.path])
+            mod = re.search(r'[^/]*(?=[.][a-zA-Z]+$)', migration.path)[0]
```

Comment:
Can we make this logic a bit simpler, and just remove the `.py` extension from the file name, and use that as the module name? Something like:

```python
mod, _ = os.path.splitext(os.path.basename(migration.path))
```

Reply:
Absolutely. I haven't had time to catch up on all the tricks python has under the sleeve. That seems a more readable approach. Done!